pkg/core/options.go

```diff
@@ -17,6 +17,8 @@ type Settings struct {
 	profilingEnabled bool
 	memProfDir       string
 
+	label string
```

Comment:
Not very important, but I am still thinking a more explicit name would help here. Some suggestions: `versionedLabels`, `withVersions`, `vLabel`...

Reply:
:p just :p .  `vLabel` it is.